src/website/views/components/modal.pug

```diff
@@ -224,3 +224,64 @@ p.-text Remove <code>a-modal__header</code> and <code>a-modal__footer</code> for
         </div>
       </div>
     </div>
+
+h4 Scrollable
+p.-text
+  | If the content height is bigger than the content modal container
+  | height, we can make the content scrollable.
+h5 Requirements
+p.-text
+  | Content Modal must have the<code>max-height</code> defined.
```

Comment:
Let's add two new utility sizing classes and encourage devs to use those versus adding inline styles. `-mh--400` and `-mh--480`

Reply:
done